node/sc/sub_bridge_handler.go

```diff
@@ -342,6 +383,46 @@ func (sbh *SubBridgeHandler) LocalChainHeadEvent(block *types.Block) {
 	}
 }
 
+func (sbh *SubBridgeHandler) handleParentChainInvalidTxResponseMsg(msg p2p.Msg) error {
+	/*
+		///////////////////// KIP71 Test ///////////////////////////
+		sbh.subbridge.blockchain.Config().KIP71CompatibleBlock = big.NewInt(100)
+		headerNum := big.NewInt(101)
+		////////////////////////////////////////////////////////////
+	*/
+
+	headerNum := sbh.subbridge.blockchain.CurrentHeader().Number
+	var invalidTxs []InvalidParentChainTx
+	if err := msg.Decode(&invalidTxs); err != nil && err != rlp.EOL {
+		return errResp(ErrDecode, "msg %v: %v", msg, err)
+	}
+	txPool := sbh.subbridge.GetBridgeTxPool()
+	for _, invalidTx := range invalidTxs {
+		if tx := txPool.Get(invalidTx.TxHash); tx != nil {
+			logger.Error("A bridge tx was not executed", "err", invalidTx.ErrStr,
```

Comment:
It doesn't seem to be an error that tx is invalid and not executed and it is in the txpool. 
It may be caused by other causes, not gasprice. Is it an intended log level?

Reply:
The intention of log level choice is because of invalid tx. One of the invalid tx cases is a tx that presented a lower gasPrice than the current header's base fee. The function `handleParentChainInvalidTxResponseMsg` handles invalid txs which were rejected by `AddRemote()` function.